samples/azure/custom-audit-transport/sample.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Custom audit transport
+summary: Azure Service Bus endpoint with audit via Azure Storage Queue
+reviewed: 2025-05-28
+component: Core
+related:
+ - transports/azure-service-bus
+ - transports/azure-storage-queues
+ - nservicebus/pipeline/features
+ - nservicebus/pipeline/manipulate-with-behaviors
+---
+
+This sample demonstrates how an endpoint can use a different transport for audting to its main transport.
+In this instance, Azure Service Bus is the main transport used by the endpoint, and Azure Storage Queues is used as the transport for audit messages.
+
+> [!WARNING]
+> Two different transports are being used, which means the Azure Storage Queue transport dispatcher does not participate in the handler transaction. Hence there could be a scenario where the audit message is successfully sent, but an error occurs later in the pipeline that causes the receiver operation to be rolled back. This would result in two conflicting instances of the same message being visible in the Service Pulse [all messages view](/servicepulse/all-messages.md), one showing as an error and the other as successfully processed.
+
+## Prerequisites
+
+Ensure that:
+
+- an instance of the latest [Azurite Emulator](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio%2Cblob-storage) is running
+- the Azure Service Bus connection is stored in an environment variable called `AzureServiceBus_ConnectionString`
+- the `audit` queue exists in Azure Storage Queues
+
+### Optional
+
+To see how the auditing messages appear in ServicePulse, two ServiceControl instances will need to be setup.
```

Comment:
Could we provide here a docker compose file with these two instances to make it easier for customers to set them up?

Reply:
Docker compose file has been added